docs/src/token.mdx

```diff
@@ -80,46 +90,204 @@ Hardware Wallet URL (See [URL spec](https://docs.solana.com/wallet-guide/hardwar
 $ solana config set --keypair usb://ledger/
 ```
 
+  </TabItem>
+  <TabItem value="jsx" label="JS">
+
+#### Yarn
+```console
+yarn add @solana/spl-token
+```
+#### npm
+```console
+npm install @solana/spl-token
+```
+#### Browser bundle
+```html
+<script src="https://unpkg.com/@solana/spl-token@latest/lib/index.life.min.js"></script>
+```
+### Configuration
+You can connect to different clusters using `Connection` in `@solana/web3.js`
+```typescript
+const web3 = require('@solana/web3.js');
+const connection = new web3.Connection(web3.clusterApiUrl('devnet'), 'confirmed');
+```
+### Keypair
+You can either get your keypair using [`Keypair`](https://solana-labs.github.io/solana-web3.js/classes/Keypair.html) from `@solana/web3.js`, or let the user's wallet handle the keypair and use `sendTransaction` from [`wallet-adapter`](https://github.com/solana-labs/wallet-adapter)
+
+  </TabItem>
+</Tabs>
+
+
+
 #### Airdrop SOL
 
 Creating tokens and accounts requires SOL for account rent deposits and
 transaction fees. If the cluster you are targeting offers a faucet, you can get
 a little SOL for testing:
+
+<Tabs className="unique-tabs" groupId="language-selection">
+  <TabItem value="cli" label="CLI" default>
+
 ```console
 $ solana airdrop 1
 ```
 
+  </TabItem>
+  <TabItem value="jsx" label="JS">
+
+```jsx
+import { clusterApiUrl, Connection, Keypair, LAMPORTS_PER_SOL } from '@solana/web3.js';
+
+const payer = Keypair.generate();
+
+const connection = new Connection(
+  clusterApiUrl('devnet'),
+  'confirmed'
+);
+
+const airdropSignature = await connection.requestAirdrop(
+  payer.publicKey,
+  LAMPORTS_PER_SOL,
+);
+
+await connection.confirmTransaction(airdropSignature);
+```
+
+  </TabItem>
+</Tabs>
+
 ### Example: Creating your own fungible token
 
+<Tabs className="unique-tabs" groupId="language-selection">
+  <TabItem value="cli" label="CLI" default>
+
 ```console
 $ spl-token create-token
 Creating token AQoKYV7tYpTrFZN6P5oUufbQKAUr9mNYGe1TTJC9wajM
 Signature: 47hsLFxWRCg8azaZZPSnQR8DNTRsGyPNfUK7jqyzgt7wf9eag3nSnewqoZrVZHKm8zt3B6gzxhr91gdQ5qYrsRG4
 ```
 
+  </TabItem>
+  <TabItem value="jsx" label="JS">
+
+```jsx
+import { createMint } from '@solana/spl-token';
+import { clusterApiUrl, Connection, Keypair, LAMPORTS_PER_SOL } from '@solana/web3.js';
+
+const payer = Keypair.generate();
+const mintAuthority = Keypair.generate();
+const freezeAuthority = Keypair.generate();
+
+const connection = new Connection(
+  clusterApiUrl('devnet'),
+  'confirmed'
+);
+
+const mint = await createMint(
+  connection,
+  payer,
+  mintAuthority.publicKey,
+  freezeAuthority.publicKey,
+  9
```

Comment:
I was trying to match exactly how it would be for the CLI. Is there a specific reason for the default to 9 on the CLI?

Otherwise my comment would be about the default for CLI, which doesn't seem too helpful.

Reply:
The CLI likely defaults to 9 because SOL has 9 decimal places.  Either way, some sort of comment will be helpful!

Reply:
Updated with comment